README.md

```diff
@@ -7,7 +7,7 @@ Users can deploy Harvester Load Balancer in any other Kubernetes clusters withou
 ## Deploying
 ```
 # helm v3
-helm install deploy/harvester-load-balancer -n harvester-system
+helm install deploy/harvester-load-balancer -n harvester-system --set <required-parameters>
```

Comment:
specify the required parameters as a good example,  e.g --set interface=eth0, address=172.16.0.x